Vlad_B/classes.py

```diff
@@ -0,0 +1,75 @@
+class Aircraft(object):
+    def __init__(self, prod_year, max_ceiling, max_speed, country, lifetime):
+        self.prod_year = prod_year
+        self.max_ceiling = max_ceiling
+        self.max_speed = int(max_speed.strip(r'km/h'))
+        self.country = country
+        self.lifetime = lifetime
+
+    @staticmethod
+    def class_info():
+        print("Class object named 'Aircraft' which contains 5 attributes: year of fabrication, maximum ceiling, maximum speed, country of origin, and lifetime")
+
+    def in_use(self):
+        if 2020 - self.prod_year < self.lifetime:
+            print("The aircraft is still in use")
```

Comment:
try to return the string instead of printing. You can print the result after the function call:
`print(P51.in_use())`. Judging by the name of the method I think a boolean value will be the best choice here. In this way, you can use the method for other use cases not only printing

Reply:
Indeed! Replaced print() with return instead